docusaurus/docs/developer-docs/brizy-internals.md

```diff
@@ -3,3 +3,37 @@ sidebar_position: 2
 ---
 
 # Brizy Internals
+
+Brizy operates on a core set of technologies we call the internals. In this section, we'll guide you through the key concepts necessary for developing addons for Brizy.
+
+# Available "Internals"
+
+Below is a list of Brizy internals along with a short description and links to more in-depth information:
+
+[//]: # (TODO: Link for editor)
+- [Brizy Editor](/docs-internals/brizy-editor/introduction)<br/>
+The editor is where users create pages and control their settings. It is divided into preview and panel areas.
+The panel hosts the list of widgets and is where users control the settings. The preview screen is where they see how the page will look.
+
+[//]: # (TODO: Link for toolbar)
+- [Toolbar](/docs-internals/toolbar)<br/>
+The toolbar is used for adding options to each widget, allowing for easy customization and configuration.
+ 
+[//]: # (TODO: Link for sidebar)
+- [Sidebar](/docs-internals/sidebar)<br/>
+  There are two sidebars in Brizy: Left Sidebar and Right Sidebar. 
+  - The right sidebar allows users to set up effects and styling for widgets in the `Editor`.
+  - The left sidebar allows users to select widgets, apply global styling, change the view, and more.
+
+- [CSS](/docs-internals/control-arguments/css)<br/>
+Developers need to know how CSS works in Brizy. This mechanism allows users to easily adjust various CSS properties via two approaches: using selector and style. On this page, you can deeply understand how CSS works in Brizy.
+
+[//]: # (TODO: Link for defaultValue)
+- [Default Values](/docs-internals/default-value)<br/>
+Each element in Brizy has its own default values, which provide predefined settings for a consistent starting point.
+
+- [Controls](/docs-internals/editor-controls/introduction)<br/>
+The "Controls" section provides detailed information about the various controls available in the Brizy editor.
+These controls allow users to input data, which can be used to customize and enhance their websites. In this section,
+you'll find descriptions, parameters, and examples for each control, helping you understand how to use and configure
+them effectively.
```

Comment:
This section provides detailed information about all controls available in the Brizy editor. These controls allow users to input data, which can be used to customize and enhance their websites. In this section,
you'll find descriptions, parameters, and examples for each control, helping you understand how to use and configure
them effectively.

P.S. was changed only first sentence + check if is gramatically correct